.github/workflows/tag_image_push_rhel.yml

```diff
@@ -126,6 +122,14 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: 'us-east-1'
 
+      - name: Get Secrets
+        uses: aws-actions/aws-secretsmanager-get-secrets@v2
+        with:
+          secret-ids: |
+            OCP_LOGIN_USERNAME,CN/OCP_USERNAME
```

Comment:
Do we need to mask ENVs as per https://hazelcast.atlassian.net/browse/DI-133?focusedCommentId=104227?

Reply:
They are already masked by the action

See https://github.com/hazelcast/hazelcast-docker/actions/runs/11479172818/job/31944918031 where I try to print or debug log them and fail.

Reply:
great. thanks for confirming.

Reply:
I am sure you will do this but delete secrets in repo after merge

Reply:
> I am sure you will do this but delete secrets in repo after merge

added to description.

Reply:
> I am sure you will do this but delete secrets in repo after merge

What about maintenance branches? They need these secretes as well

Reply:
> > I am sure you will do this but delete secrets in repo after merge
> 
> What about maintenance branches? They need these secretes as well

Once the PR is backported to all maintenance branches, they wont